packages/oasgraph/test/example_api.test.js

```diff
@@ -734,4 +734,95 @@ test('Option provideErrorExtensions should prevent error extensions from being c
       })
     })
   })
+})
+
+test('Option customResolver', () => {
+  let options = {
+    customResolvers: {
+      'Example API': {
+        '/users/{username}': {
+          'get': () => {
```

Comment:
Let's make sure somwhere that the right arguments are passed to the custom resolver functions.